src/process.ts

```diff
@@ -115,6 +119,12 @@ export class PowerShellProcess {
             };
         }
 
+        // Enables Hot Reload in .NET for the attached process
+        // https://devblogs.microsoft.com/devops/net-enc-support-for-lambdas-and-other-improvements-in-visual-studio-2015/
+        if (this.devMode) {
+            (envMixin as Record<string,string>).COMPLUS_FORCEENC = "1";
```

Comment:
Woah cool. And glad I wired up the environment mixin.

Reply:
yeah in javascript objects are always mutable, so this is a pseudo-hack to add in the addition env var. The better way would be to make it a Record from the beginning, but it's not that big of a deal.